test/grpc/test_nexus.js

```diff
@@ -685,6 +685,32 @@ describe('nexus', function () {
       });
     });
 
+    it('should change ANA state to inaccessible', (done) => {
```

Comment:
Is it possible to check that the new state has been actually set ?

Reply:
Done with the getter.